components/Topic.js

```diff
@@ -1,7 +1,21 @@
 import React from "react";
-
-const Topic = () => {
-	return <div></div>;
+const Topic = ({ topic, backgroundColor, textColor }) => {
+	return (
+		<div>
+			<div
+				className='flex justify-center items-center w-96 h-36 rounded-3xl border-2 border-black mb-8 shadow-black'
```

Comment:
Height and width shouldn't be hard coded with w-96 and h-36, it should just take up space of the column it is in